ui/desktop/app/routes/scopes/scope/authenticate.js

```diff
@@ -28,13 +29,42 @@ export default class ScopesScopeAuthenticateRoute extends Route {
    * scope and all scopes (for org navigation).
    * @return {Promise} `{scope, scopes, authMethods}`
    */
-  model() {
+  async model() {
     const { id: scope_id } = this.modelFor('scopes.scope');
 
+    // Fetch auth methods for current scope
+    const authMethods = await this.resourceFilterStore.queryBy(
+      'auth-method',
+      {
+        authorized_actions: [{ contains: 'authenticate' }],
+      },
+      {
+        scope_id,
+      },
+    );
```

Comment:
Is this better than just doing a `peekAll` and filtering the relevant auth methods since we already retrieve all auth methods below?

Reply:
Updated.